helm/cluster/files/apps/observability-bundle.yaml

```diff
@@ -11,3 +11,44 @@ namespace: kube-system
 # used by renovate
 # repo: giantswarm/observability-bundle
 version: 1.12.0
+defaultValues:
+  kubeStateMetricsCustomResources:
+    {{- if include "cluster.internal.hasKarpenterNodePool" $ }}
+    karpenter_machine_v1: true
```

Comment:
This should be aws only right?

Reply:
For now yes, but potentially karpenter could be deployed on other providers like Azure. This check is useless atm, until [this PR](https://github.com/giantswarm/cluster-aws/pull/1038) is merged, since we don't deploy karpenter as a cluster app yet. So I think I'll remove it for now and add it as part of the karpenter integration effort later on

Reply:
You can keep it it's fine to me as long as it does not screw up ksm :)

Reply:
well since the karpenter integration is not yet final, I'm not 100% sure the check will work as it is anyway. So I prefer to just remove it for now